templates/case-studies/index.html

```diff
@@ -0,0 +1,4 @@
+{% extends "case-studies/base_case-studies.html" %}
+
+{% block title %}Case Studies{% endblock %}
```

Comment:
btw Britney, doesn't it make sense to eventually create this page as a list of all the case studies? Similar to how we show engage pages at ubuntu.com/engage or tutorials at ubuntu.com/tutorials. It's not something we should do now. Maybe worth asking Gloria.
Individual case studies will be shown at ubuntu.com/case-studies/<case_study_name_or_title>.

Reply:
Thanks for the review, Gloria has mentioned something similar to your suggestion for them to be stored in  `ubuntu.com/case-studies/<case_study_name_or_title>`. Perhaps I can create another PR once all the sections are done to remove the example from the index page and to an individual page instead.